index.js

```diff
@@ -8,8 +8,51 @@ const http = require('http'),
         The time is: ${rightNow}
       </p>
     </body>
+ 
+      <body>
+   <div style="position:  absolute; bottom: 5px; background-color: white">
+<h2> Characters in The Lord of The Rings </h2>
+ <ul>
+   <li>   <b> Frodo Baggins </b>  </li>
+   <li>   <b> Bilbo Baggins </b>  </li>
+   <li>   <b> Gandalf '<i> (the Grey, later the White) </i>' </b> </li>
```

Comment:
`<em>` might be more appropriate here since you are providing emphasis, See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em